### PR TITLE
Mat 8826 justification remapped to display element

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE FHIR SERVICE

Jira Ticket: [MAT-8826](https://jira.cms.gov/browse/MAT-8826)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/madiefhirservice/services/HumanReadableService.java [hunk 1]
```diff
@@ -70,7 +71,13 @@ private void escapeTopLevelProperties(org.hl7.fhir.r5.model.Measure measure) {
           measure.getRelatedArtifact().stream()
               .map(
                   relatedArtifact ->
-                      relatedArtifact.setCitation(escapeStr(relatedArtifact.getCitation())))
+                      relatedArtifact
+                              .getType()
+                              .toCode()
+                              .equals(RelatedArtifactType.JUSTIFICATION.toCode())
+                          ? relatedArtifact.setDisplayElement(
+                              new StringType((relatedArtifact.getDisplayElement().toString())))
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/madie/madiefhirservice/services/HumanReadableService.java]: Is the displayElement user inputted or the display value from the terminology?

Comment (sb-cecilialiu) [reply src/main/java/gov/cms/madie/madiefhirservice/services/HumanReadableService.java]: @jkotanchik-SB user inputted

Comment (jkotanchik-SB) [reply src/main/java/gov/cms/madie/madiefhirservice/services/HumanReadableService.java]: Does it need to be escaped, then?

Comment (sb-cecilialiu) [reply src/main/java/gov/cms/madie/madiefhirservice/services/HumanReadableService.java]: @jkotanchik-SB Yes, it should. I just updated it. Thanks Joe!

**Review by jkotanchik-SB — APPROVED** [review]
